v2/internal/testutil/testutil.go

```diff
@@ -0,0 +1,84 @@
+package testutil
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+
+	r "github.com/stretchr/testify/require"
+)
+
+func Main(m *testing.M) {
+	// never test against live key.
+	pkey, skey := Keys()
+	switch {
+	case pkey == "" || skey == "":
+		log.Fatalln("no test key specified, please set both $OMISE_PKEY and $OMISE_SKEY")
+	case !strings.HasPrefix(pkey, "pkey_test_") || !strings.HasPrefix(skey, "skey_test_"):
+		log.Fatalln("specified key is invalid or is not a test key!!! You might lose money!!!")
+	}
+
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
+func LogObj(t *testing.T, obj interface{}) {
+	t.Log(fmt.Sprintf("%#v", obj))
+}
+
+func Require(t *testing.T, env string) {
+	env = strings.ToUpper(env)
+	val := strings.ToLower(os.Getenv(env))
+
+	switch val {
+	case "", "0", "false", "no":
+		t.Skip("requires " + env + "=1")
+	}
+}
+
+func AssertJSONEquals(t *testing.T, m1 map[string]interface{}, m2 map[string]interface{}) {
+	assertJSONEquals(t, "", m1, m2)
+}
+
+func isMap(v interface{}) bool {
+	_, ok := v.(map[string]interface{})
+	return ok
+}
+
+func assertJSONEquals(t *testing.T, prefix string, m1 map[string]interface{}, m2 map[string]interface{}) {
```

Comment:
there's https://pkg.go.dev/github.com/stretchr/testify/require?tab=doc#JSONEq for this kind of assertion and it's already imported here. could be use that instead